public/pages/Correlations/containers/CreateCorrelationRule.tsx

```diff
@@ -415,43 +410,37 @@ export const CreateCorrelationRule: React.FC<CreateCorrelationRuleProps> = (
                       </>
                     );
                   })}
-                  {disableForm ? null : (
-                    <EuiButton
-                      style={{ width: 125 }}
-                      onClick={() => {
-                        props.setFieldValue(`queries[${queryIdx}].conditions`, [
-                          ...query.conditions,
-                          ...correlationRuleStateDefaultValue.queries[0].conditions,
-                        ]);
-                      }}
-                      iconType={'plusInCircle'}
-                      disabled={disableForm}
-                    >
-                      Add field
-                    </EuiButton>
-                  )}
+                  <EuiButton
+                    style={{ width: 125 }}
```

Comment:
What happens with scaling? Will this have problems on small screens like laptops?

Reply:
No issue since it is the only button on the line and is small enough that any screen should be able to fit it